MiscVBAFunctions/Modules/MiscOs.bas

```diff
@@ -74,58 +74,48 @@ Private Function parentDir(ByVal folder)
 End Function
 
 
-Public Sub CreateFolders(ByVal strPath As String, _
-                  Optional doShell As Boolean = False)
+Public Function MakeDirs(ByVal StrPath As String) As folder
```

Comment:
Can we add the `ExistOk=True` parameter? To make it more like the python equivalent, and be able to replicate the wanted behaviour when `ExistOk=False`.

We are deviating from the Python default. Do you think that is a problem @BartJan142 , @heetbeet ?